src/flare_pp/bffs/sparse_gp.cpp

```diff
@@ -1190,19 +1135,8 @@ void SparseGP ::set_hyperparameters(Eigen::VectorXd hyps) {
                + 1 / (force_noise * force_noise) * f_noise_one 
                + 1 / (stress_noise * stress_noise) * s_noise_one; 
 
-  t2 = std::chrono::high_resolution_clock::now();
-  duration = (double) std::chrono::duration_cast<std::chrono::milliseconds>( t2 - t1 ).count();
-  std::cout << "Time: set_hyp: update noise " << duration << std::endl;
-  t1 = std::chrono::high_resolution_clock::now();
-
   // Update remaining matrices.
   update_matrices_QR();
-
-  t2 = std::chrono::high_resolution_clock::now();
-  duration = (double) std::chrono::duration_cast<std::chrono::milliseconds>( t2 - t1 ).count();
-  std::cout << "Time: set_hyp: update qr " << duration << std::endl;
-  t1 = std::chrono::high_resolution_clock::now();
-
```

Comment:
Can consider timing the SGP likelihood in the dev directory (future PR)

Reply:
added to the "Todos" in the description